### ISSUE TITLE
Search: one article showing with "published on Invalid Date" in Safari and Firefox

### ISSUE THREAD

**Issue by sianroderick** [elifesciences/lens-browser#31]
On http://cdn.substance.io/lens-browser/ in Firefox or Safari, search for "Activity is a slave to many masters".  The "published on" date says "Invalid Date" on the search results page.  The date should match that shown within the article (http://lens.elifesciences.org/06351/) of "Thu, 12 Feb 2015".

**Comment by michael** [comment]
This is because of an inconsistent date format specified in the XML which can not be parsed by our indexer and Safari. This is also the reason why this article is listed before newer articles Because `"12-2-2015"` > `"26-03-2015"`. 

```xml
<pub-date publication-format="electronic" date-type="pub">
<day>12</day>
<month>2</month>
<year>2015</year>
</pub-date>
```

Month should be `02`. Can this be fixed somehow in the XML?

**Comment by sianroderick** [comment]
Thanks - I'll get the production team to take a look and get the XML fixed.

**Comment by michael** [comment]
Thanks :)

Maybe you could also ask if there would be a way to cleanup instance organism values for instance (also other facets), as sometimes they have a capital letter, sometimes not. 

![image](https://cloud.githubusercontent.com/assets/2931/7249159/9a09758c-e818-11e4-8f6f-72dac3e5f5cf.png)

The ideal place to normalize this would be in the source XML, so we don't need to develop a sanitizer.

**Comment by Melissa37** [comment]
Agree! Sian and I have chatted about that - we need to do a clean up of all keywords. It will probably be tied in with the other work going on, and so would be live in Nov time.
